.editorconfig

```diff
@@ -6,19 +6,18 @@ root = true
 
 [*]
```

Comment:
can't see information about real test execution. Please check "test" section in your build.gradle. Expected something like this TestUnitTest > testname PASSED